fix portal T&C's

olhabn opened the pull request:
Terms and Conditions link was opened in modal window. It was changed to opening link in the current window

Address: https://nanopay.atlassian.net/browse/CPF-2818

Commit:
Terms and Conditions link was opened in modal window. It was changed to opening link in the current window

kgrgreer commented on src/foam/nanos/u2/navigation/FooterView.js:
```diff
@@ -137,7 +137,7 @@ foam.CLASS({
       label: '',
       code: function(X) {
-        X.openTermsModal();
+        this.window.location.assign(X.appConfig.termsAndCondLink);
```
Better to do X.window.location because it allows the application to work with multiple windows.

olhabn replied:
Fixed. https://github.com/foam-framework/foam2/pull/2649/commits/4a2bd6b91d456f279ec30f19f193ba9f3adb4e4d

Commit:
Opening link with context

olhabn commented:
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ffoam-framework%2Ffoam2%2Fpull%2F2649) for more info**.
@googlebot I fixed it.
